src/main/java/com/ericsson/gerrit/plugins/eiffel/listeners/PatchsetCreatedEventListener.java

```diff
@@ -64,8 +64,9 @@ protected void prepareAndSendEiffelEvent(Event gerritEvent,
         EiffelSourceChangeCreatedEvent eiffelEvent = EiffelSourceChangeCreatedEventGenerator.generate(
                 patchSetCreatedEvent, pluginConfig);
         EiffelEventSender eiffelEventSender = new EiffelEventSender(pluginConfig);
-        eiffelEventSender.send(eiffelEvent);
-
+        eiffelEventSender.setType("EiffelSourceChangeCreatedEvent");
```

Comment:
Please remove the hardcoded type here. Type is the same as the Event Class. One could use EventClass....class.getName() or something.

Or move the type to a static variable.

Reply:
Ah true, I think it's getSimpleName otherwise you get the package name as well, but yes we could do that.